docs/configuration/network_filters/tcp_proxy_filter.rst

```diff
@@ -11,14 +11,13 @@ TCP proxy :ref:`architecture overview <arch_overview_tcp_proxy>`.
     "type": "read",
     "name": "tcp_proxy",
     "config": {
-      "cluster": "...",
-      "stat_prefix": "..."
+      "stat_prefix": "...",
+      "route_config": "{...}"
     }
   }
 
-cluster
-  *(required, string)* The :ref:`cluster manager <arch_overview_cluster_manager>` cluster to connect
-  to when a new downstream network connection is received.
+:ref:`route_config <config_network_filters_tcp_proxy_route_config>`
+  *(required, object)* The route table for the filter. All filter instances must have a route table, even if it is empty.
```

Comment:
In the interest of brevity, I suggest merging the route_config with this document and linking to the _route doc from here as well.